wikipron/extract/core.py

```diff
@@ -35,7 +36,7 @@ def yield_pron(
         if _skip_pron(pron, config):
             continue
         try:
-            pron = config.process_pron(pron)
+            pron = config.process_pron(unicodedata.normalize("NFD", pron))
```

Comment:
Maybe do this in two steps:

   pron = unicodedata.normalize("NFD", pron)
   pron = config.process_pron(pron)

Don't know why but I think that might be easier to read.